### ISSUE TITLE
[FEATURE] MiningSpoof

### ISSUE THREAD

**Issue by HermanGoring** [John200410/rusherhack-issues#836]
**What feature would you like to be added.**
Mine without your pickaxe in hand (has to be in hotbar)

**Is your feature request related to a problem? Please describe.**
No.

**Additional context**
https://github.com/bebeli555/AutoBot/blob/main/src/main/java/me/bebeli555/autobot/mods/other/MiningSpoof.java

**Comment by jumboman32** [comment]
What is the use of this?

**Comment by HermanGoring** [comment]
> What is the use of this?

it would be useful if i was trapped and did not have to switch to a pickaxe in the midst of crystal pvp

**Comment by DiscordBotTester** [comment]
> > What is the use of this?
> 
> it would be useful if i was trapped and did not have to switch to a pickaxe in the midst of crystal pvp

offhand is a thing yk? just put crystals in offhand and pick in mainhand

**Comment by system69-e** [comment]
> > > What is the use of this?
> > 
> > 
> > it would be useful if i was trapped and did not have to switch to a pickaxe in the midst of crystal pvp
> 
> offhand is a thing yk? just put crystals in offhand and pick in mainhand

ever heard of a non cringe server with offhand nerf

**Comment by HarryPotter42MC** [comment]
packetmine can do that just click a blocks and when its green switch to pick and what you say wont work and learn hotkeying
